src/main/py/hl7_listener/messaging/sqs.py

```diff
@@ -0,0 +1,54 @@
+from typing import (
+    Union,
+    Callable,
+)
+
+from aiobotocore.session import get_session
+
+from covera_ddtrace import inject_ddtrace
+
+from hl7_listener.utils import (
+    dd_utils,
+    logger_util,
+    logging_codes
+)
+import hl7_listener.messaging.settings as msgr_config
+from hl7_listener.messaging.base import MessagingInterface
+
+
+logger = logger_util.get_logger(__name__)
+
+
+class SQSMessager(MessagingInterface):
+    @inject_ddtrace
+    async def connect(self) -> bool:
+        """Create an aiobotocore session if we don't have one."""
+        self.conn = get_session()
+
+    @inject_ddtrace
+    async def send_msg(self, msg: Union[str, bytes], send_as: Callable = str) -> None:
+        """Sends a msg to an sqs queue."""
+        assert send_as in (str, bytes)
+        assert isinstance(msg, (str, bytes))
+
+        to_send = msg
+        if isinstance(msg, str) and send_as == bytes:
+            to_send = msg.encode()
```

Comment:
I think you'll get `botocore.exceptions.ParamValidationError` on this since `MessageBody` below  must be a string right? So I think in both cases we will need to send as a string

Reply:
Addressed in latest for both SQS and NATS messagers.